lib/ToolbarPanelHub.jsm

```diff
@@ -34,6 +34,9 @@ class _ToolbarPanelHub {
     this._getMessages = getMessages;
     if (this.whatsNewPanelEnabled) {
       this.enableAppmenuButton();
+      // TODO: this will eventually be called when the badge message gets triggered
+      // instead of here in init.
+      this.enableToolbarButton();
```

Comment:
Was this added back by accident? It should now be trigger by the badge PR that landed in the `whatsnew-panel` branch

Reply:
gah!